lib/git_reflow/commands/setup.rb

```diff
@@ -15,11 +15,25 @@
     end
 
     choose do |menu|
-      menu.header = "Available remote Git Server services:"
+      menu.header = "Available remote Git Server services"
       menu.prompt = "Which service would you like to use for this project?  "
 
       menu.choice('GitHub')    { GitReflow::GitServer.connect reflow_options.merge({ provider: 'GitHub', silent: false }) }
       menu.choice('BitBucket (team-owned repos only)') { GitReflow::GitServer.connect reflow_options.merge({ provider: 'BitBucket', silent: false }) }
     end
+
+    choose do |lgtm_menu|
+      lgtm_menu.header = "LGTM Options"
```

Comment:
Can we update the wording here to `Approval Options` and reference `approvals` rather than `lgtm`.  Since we allow for an approval regex I think this may be confusing.